src/navigation/Nav.jsx

```diff
@@ -325,7 +327,7 @@ export class Nav extends React.Component {
 				),
 				onClickAction:
 					media.isAtMediumUp && !isGroupsLoaded
-						? profile.getSelfGroupsQuery
+						? (profile.getSelfGroupsQuery, profile.getSelfGroupDraftQuery)
```

Comment:
this syntax doesn't look right - should only be one function, not two inside parens?